pkg/sql/opt/exec/explain/output.go

```diff
@@ -372,22 +372,23 @@ func (ob *OutputBuilder) AddKVReadStats(rows, bytes int64) {
 
 // AddKVTime adds a top-level field for the cumulative time spent in KV.
 func (ob *OutputBuilder) AddKVTime(kvTime time.Duration) {
-	ob.AddRedactableTopLevelField(RedactVolatile, "cumulative time spent in KV", humanizeutil.Duration(kvTime))
+	ob.AddRedactableTopLevelField(
```

Comment:
Don't you want AddTopLevelField since kvTime is now safe?

Anyway no idea how this works, better to leave a TODO and ping someone from SQL to follow up.

Reply:
ditto for some others here.

Reply:
The redaction here seems to be used for a different purpose, i.e. to mask fields that change between runs (note `RedactVolatile`). I assume this is for e.g. testing purposes, to avoid spurious diffs and such.